environment_setup/setup_software.sh

```diff
@@ -54,6 +54,7 @@ host_software_packages=(
     protobuf-compiler # This is required for the "NanoPb" library, which does not
                       # properly manage this as a bazel dependency, so we have 
                       # to manually install it ourselves
+    kcachegrind # This lets us view the profiles output by callgrind
```

Comment:
should we also explicitly install valgrind?

Reply:
Already done: https://github.com/UBC-Thunderbots/Software/blob/master/environment_setup/setup_software.sh#L74

Reply:
Ah didn't see it in snap